test/e2e/apps/job.go

```diff
@@ -513,6 +513,124 @@ done`}
 		gomega.Expect(gotIndexes).To(gomega.Equal(wantIndexes), "expected completed indexes %s, but got %s", wantIndexes, gotIndexes)
 	})
 
+	/*
+		Testcase: Ensure that job with successPolicy succeeded when all indexes succeeded
+		Description: Create an indexed job with successPolicy.
+		Verify that job got SuccessCriteriaMet with SuccessPolicy reason and Complete condition
+		once all indexes succeeded.
+	*/
+	ginkgo.It("with successPolicy should succeeded when all indexes succeeded", func(ctx context.Context) {
+		parallelism := int32(1)
+		completions := int32(1)
```

Comment:
IMO, it would be better to have at least 2 pods here.

Reply:
Sounds good to me.

Reply:
Done.